html/pfappserver/lib/pfappserver/PacketFence/Controller/Graph.pm

```diff
@@ -466,6 +467,7 @@ sub dashboard :Local :AdminRole('REPORTS') {
     }
     $c->stash->{graphs} = $graphs;
     $c->stash->{cluster} = pf::cluster::members_ips();
+    $c->stash->{nodecategories} = [pf::nodecategory::nodecategory_view_all()];
```

Comment:
we call this roles now even though the class is nodecategory

So that needs to be renamed here and in the template